server/jetstream_api.go

```diff
@@ -499,36 +483,31 @@ const JSApiMsgGetResponseType = "io.nats.jetstream.api.v1.stream_msg_get_respons
 // JSWaitQueueDefaultMax is the default max number of outstanding requests for pull consumers.
 const JSWaitQueueDefaultMax = 512
 
-// JSApiConsumerCreateResponse.
 type JSApiConsumerCreateResponse struct {
 	ApiResponse
 	*ConsumerInfo
 }
 
 const JSApiConsumerCreateResponseType = "io.nats.jetstream.api.v1.consumer_create_response"
 
-// JSApiConsumerDeleteResponse.
 type JSApiConsumerDeleteResponse struct {
 	ApiResponse
 	Success bool `json:"success,omitempty"`
 }
 
 const JSApiConsumerDeleteResponseType = "io.nats.jetstream.api.v1.consumer_delete_response"
 
-// JSApiConsumerInfoResponse.
```

Comment:
these are more or less useless comments so just remove them, they dont even make my IDE not complain

Reply:
What about `go vet` and `staticcheck`?

Reply:
Used that linters on my IDE would complain, but not any more. go vet and staticcheck don't seem to have a problem with those either. So agreed that if we are not adding more meaningful comment, then better removed.